doc/changelog.md

```diff
@@ -61,6 +61,9 @@
   `__quantum__rt__device_init(int8_t *, int8_t *, int8_t *)`.
   [(#396)](https://github.com/PennyLaneAI/catalyst/pull/396)
 
+* Add C++ macros for generating the device entry-point and declarations.
+  [(#400)](https://github.com/PennyLaneAI/catalyst/pull/400)
+
```

Comment:
Given that the plugin system has not been released yet, you should add this PR to the existing entry.